src/main/kotlin/org/rust/lang/core/psi/BuiltinAttributes.kt

```diff
@@ -5,188 +5,94 @@
 
 package org.rust.lang.core.psi
 
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.diagnostic.logger
+import java.io.IOException
+
 
 // Bump `RsFileStub.Type.STUB_VERSION` if modified
```

Comment:
Now it's almost impossible to recall it. For example, you forgot to do it - the previous list contained 158 attributes (excluding stdlib proc macros), the current one contains 162

Reply:
It would be nice to automate updating of `builtin-attributes.json` and update `RsFileStub.Type.STUB_VERSION` if needed on CI

Reply:
I introduce a separate version for attributes (`RS_BUILTIN_ATTRIBUTES_VERSION`) and also I disabled the `builtin-attributes.json` generation on CI for now (to enable it, you need to pass `GENERATE_BUILTIN_ATTRIBUTES_JSON=1` env variable to the attribute-info script). So the next step is adjusting CI - someone can make `RS_BUILTIN_ATTRIBUTES_VERSION` version bumping and then enable the `builtin-attributes.json` updates